app/controllers/mock.php

```diff
@@ -234,29 +234,74 @@
     ->param('z', null, new ArrayList(new Text(256)), 'Sample array param')
     ->param('file', [], new File(), 'Sample file param', false)
     ->inject('request')
-    ->action(function ($x, $y, $z, $file, $request) {
+    ->inject('response')
+    ->action(function ($x, $y, $z, $file, $request, $response) {
         /** @var Utopia\Swoole\Request $request */
+        /** @var Utopia\Swoole\Response $response */
         
         $file = $request->getFiles('file');
```

Comment:
Can we use a different name for this variable instead of `$file` because we're using the same variable for the request param in line 238

Reply:
Also why are we not using the `$file` variable from L238 ?